spin/cmds/meson.py

```diff
@@ -14,6 +15,8 @@
 from .util import get_commands, get_config
 from .util import run as _run
 
+LCOV_OUTPUT_FILE = Path("build", "meson-logs", "coveragereport")
```

Comment:
This will depend on the build path the user specifies, so has to be set inside the build fn.